test/binmaxRight.js

```diff
@@ -0,0 +1,79 @@
+function binaryRightmost(stree, ptree, cat)
```

Comment:
Is this file even being used? I think it is leftover from the fall or winter. It's not syntactically valid.